### PR TITLE
feat: [DHIS2-11221] Program stage event list page with routing

### PR DESCRIPTION
**Pull request by huocha**

### CODE REVIEW

File: src/core_modules/capture-core/components/Pages/StageEvent/StageEventList/StageEventList.component.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import type { Props } from '../StageEventListPage.types';
+
+const getStyles = () => ({});
+
+const StageEventListComponentPlain = () => (<>
+    <div>Stages and Events</div>
+</>);
+
+export const StageEventListComponent: ComponentType<$Diff<Props, CssClasses>> =
+  compose(
```
Comment (simonadomnisoru) [inline src/core_modules/capture-core/components/Pages/StageEvent/StageEventList/StageEventList.component.js:15]: Hey, 
I was wondering what is the need to use `compose` here when there is only one HOC? Looking to other components we don't use `compose` for this case. For example https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WidgetProfile/WidgetProfile.component.js#L114

Thanks!

**Comment by simonadomnisoru** [comment]
Hey, 
Can you please add some tests for the new stage event list page. 
Thanks!

File: src/core_modules/capture-core/components/Pages/StageEvent/StageEventList/StageEventList.component.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import React from 'react';
+import type { ComponentType } from 'react';
+import withStyles from '@material-ui/core/styles/withStyles';
+import { compose } from 'redux';
+import type { Props } from '../StageEventListPage.types';
+
+const getStyles = () => ({});
+
+const StageEventListComponentPlain = () => (<>
+    <div>Stages and Events</div>
```
[the remark this answers is not in the document]
Comment (huocha) [reply src/core_modules/capture-core/components/Pages/StageEvent/StageEventList/StageEventList.component.js:11]: @simonadomnisoru this isn't the content of the page but a placeholder for the new page. I don't think we need i18n

File: src/core_modules/capture-core/components/Pages/StageEvent/StageEventList/StageEventList.component.js (hunk 1 above)
Comment (huocha) [reply src/core_modules/capture-core/components/Pages/StageEvent/StageEventList/StageEventList.component.js:15]: Hey, I plan to add a HOC to handle error and loading later on. But okay to remove

**Comment by huocha** [comment]
@simonadomnisoru  thanks for the feedback! I added the tests and adjustment :)

**Review by simonadomnisoru — APPROVED** [review]

**Review by JoakimSM — APPROVED** [review]
LGTM 👍

**Pull request merged**